include/neml2/tensors/BatchTensor.h

```diff
@@ -266,34 +271,97 @@ BatchTensor<N>::operator-() const
   return -torch::Tensor(*this);
 }
 
+template <TorchSize N>
+BatchTensor<N>
+BatchTensor<N>::pow(const BatchTensor<N> & n) const
+{
+  return torch::pow(*this, n);
+}
+
 template <TorchSize N>
 BatchTensor<N>
 BatchTensor<N>::identity(TorchSize n, const torch::TensorOptions & options)
 {
   return torch::eye(n, options).index(TorchSlice(N, torch::indexing::None));
 }
 
+template <TorchSize N>
+BatchTensor<N>
+operator+(const BatchTensor<N> & a, const Real & b)
+{
+  return torch::operator+(a, b);
+}
+
+template <TorchSize N>
+BatchTensor<N>
+operator+(const Real & a, const BatchTensor<N> & b)
+{
+  return torch::operator+(a, b);
+}
+
```

Comment:
I had to add a bunch of Real overloads like this one. So that we don't do the stupid scalar.to(tensor) thing everywhere.